scripts/pi-hole/php/teleporter.php

```diff
@@ -540,7 +540,7 @@ function process_file($contents)
 }
 else
 {
-	$tarname = "pi-hole-teleporter_".date("Y-m-d_h-i-s").".tar";
+	$tarname = "pi-hole-teleporter_".date("Y-m-d_H-i-s").".tar";
```

Comment:
In fact, if you could split this patch too to a separate PR, we could get that merged sooner.